### PR TITLE
Add a CI job that checks that the example in `README.md` matches the `example.yml` file

### PR DESCRIPTION
**Pull request by DilumAluthge**
Currently, the example TagBot workflow is defined in two different sources in this repository:
1. The `example.yml` file
2. The `Setup` section of the `README.md`

Therefore, there is a risk that a change is made to one of the two sources but is not made to the other sources, resulting in the two sources getting out of sync.

This PR adds a simple CI check to make sure that the example in `README.md` matches the `example.yml` file.

### CODE REVIEW

**Comment by DilumAluthge** [comment]
Bump @christopher-dG

File: .github/workflows/check_readme.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  readme:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: '1'
+      - run: julia --color=yes --project=.ci -e 'import Pkg; Pkg.instantiate()'
+      - run: julia --color=yes --project=.ci -e 'import Pkg; Pkg.precompile()'
+      - run: julia --color=yes --project=.ci -e 'include(".ci/check_readme.jl"); check_readme(pwd())'
```
Comment (christopher-dG) [inline .github/workflows/check_readme.yml:21]: I feel like we can just keep this minimal with Bash
```suggestion
      - run: |
          readme="$(<README.md)"
          example="$(echo -e "\`\`\`yml\n$(<example.yml)\n\`\`\`")"
          [[ "$readme" == *"$example"* ]]
```

File: .github/workflows/check_readme.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: Check README
```
Comment (christopher-dG) [inline .github/workflows/check_readme.yml:1]:
```suggestion
name: Compare example workflows
```

**Review by christopher-dG — CHANGES_REQUESTED** [review]
Personally I feel like shell is enough for this, but I do agree with the premise of this PR

**Pull request merged**